tuna/miopen/celery_tuning/celery_tasks.py

```diff
@@ -49,4 +49,4 @@ def celery_enqueue(context):
   logger.info("Enqueueing task %s", context['job'])
   worker = prep_worker(copy.deepcopy(context))
   ret = worker.run()
-  return ret, context
+  return {"ret": ret, "context": context}
```

Comment:
Perhaps this was not necessary, since the error seemed to stem from the sharing of databases for the broker and the result backend